scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -1519,11 +1539,28 @@ def get_sql_front50_db_conn(migrate_data_flag):
         if migrate_data_flag == 'true':
             sqldb_front50 = mysql.connector.connect(database='front50', user=spin_db_username, password=spin_db_password, host=spin_db_host, autocommit = True)
         else:
-            sqldb_front50 = mysql.connector.connect(database='front50', user=spin_db_username,
-                                                    password=spin_db_password, host=spin_db_host)
+            sqldb_front50 = mysql.connector.connect(database='front50', user=spin_db_username, password=spin_db_password, host=spin_db_host)
         print("Spinnaker front50 database connection established successfully")         
         return sqldb_front50
 
+def get_postgres_orca_db_conn():
+    if spin_db_type == 'postgres':
+        #Establishing the spinnaker orca postgres database connection       
+        postgresdb_orca = psycopg2.connect(database='orca', user=spin_db_username, password=spin_db_password, host=spin_db_host,port=spin_db_port)
+        if migrate_data_flag == 'true':
+           postgresdb_orca.autocommit = True
+        print("Spinnaker orca database connection established successfully")         
+        return postgresdb_orca.cursor()
+
+def get_postgres_front50_db_conn():
+    if spin_db_type == 'postgres':
+        #Establishing the spinnaker front50 postgres database connection       
+        postgresdb_front50 = psycopg2.connect(database='front50', user=spin_db_username, password=spin_db_password, host=spin_db_host,port=spin_db_port)
+        if migrate_data_flag == 'true':
+           postgresdb_orca.autocommit = True
```

Comment:
@sudhakaropsmx please be watchful to such copy paste mistakes :( 
this should be "postgresdb_front50.autocommit" instead of "postgresdb_orca.autocommit"

Reply:
updated